Dockerfile

```diff
@@ -349,6 +349,7 @@ RUN chmod +x /usr/local/bin/doctor && \
     chmod +x /usr/local/bin/realtime && \
     chmod +x /usr/local/bin/executor && \
     chmod +x /usr/local/bin/schedule && \
+    chmod +x /usr/local/bin/schedule-new && \
```

Comment:
Reminder to remove all `-new` stuff from this PR and deprecate the old scheduler and resque together.